oneview_redfish_toolkit/tests/api/test_zone.py

```diff
@@ -104,13 +102,12 @@ def test_spt_when_connections_are_not_configured(
             Tests Zone with no resource block for network when handling
             a Server profile template with no connections configured
         """
-
+        self.maxDiff = None
         profile_template = copy.deepcopy(self.server_profile_template)
         profile_template["connectionSettings"]["connections"] = []
 
         zone = Zone(profile_template,
-                    self.available_targets,
+                    self.server_hardwares,
```

Comment:
~self.server_hardwares~ `self.server_hardwares`. Without the "s". It does not have plural form.
Change the the others occurrencies of _hardwares_. =)